sideboard/websockets.py

```diff
@@ -46,6 +54,10 @@ def reset(cls, **kwargs):
         for key, val in kwargs.items():
             cls.set(key, val)
 
+    @class_property
+    def client_data(cls):
+        return cls.setdefault('client_data', {})
```

Comment:
This is a useful websocket-related feature.  It's basically the websocket equivalent of a web server session.